documentation/search-definitions.html

```diff
@@ -94,12 +94,13 @@ <h3 id="multivalue-fields">Multivalue fields</h3>
 </p><p>
 Most multivalue fields can be used in <a href="reference/grouping-syntax.html#multivalue-attributes">grouping</a>.
 </p><p>
-Multivalue fields and summaries - how to know which element hit
 </p><p>
+When searching in array of struct,
```

Comment:
This also applies for maps (both of primitive types and struct).

Reply:
thanks, will add map